tutorials/token-ownership-tutorial.md

```diff
@@ -286,5 +290,5 @@ To answer the question "which NFTs does this account own" you need to consult th
 
 If you trust an account and want to see which NFT it owns, look at its NamedKeys. See how many NamedKeys the account has for a given collection (in the format "m_1000_p_#"). These NamedKeys point to the "page_#" dictionaries that contain token ownership information. Using the state root hash and the dictionary address for each page (in the format "dictionary-...", you can retrieve token ownership details.
 
-Suppose you want additional security or cannot trust the account's NamedKeys. In that case, you need to interrogate the NFT contract using the "page_table" dictionary to determine which pages are allocated to the account. Then, you can use the seed URef of the page-specific dictionaries and the account hash (without the "account-hash-" prefix) to retrieve the tokens assigned to the account specified.
+Suppose you want additional security or cannot trust the account's NamedKeys. In that case, you need to explore the NFT contract using the "page_table" dictionary to determine which pages are allocated to the account. Then, you can use the seed URef of the page-specific dictionaries and the account hash (without the "account-hash-" prefix) to retrieve the tokens assigned to the account specified.
```

Comment:
I would use "query" instead of "explore"

Reply:
I like that, thanks @gRoussac!